src/components/Clusters/views/ClusterOverview/ClusterStats.js

```diff
@@ -72,6 +95,71 @@ export default function ClusterStats({ data }) {
             }}
           />
         </Card>
+        <CardWithTooltip
+          title={t('cluster-overview.statistics.namespaces-health')}
+          tooltip={{
+            content: t('cluster-overview.tooltips.namespaces-health-info'),
+            position: 'bottom',
+          }}
+          icon={'sys-help'}
+        >
+          {podsData && (
+            <ProgressIndicatorWithPercentage
+              title={t('cluster-overview.statistics.healthy-pods')}
+              value={calculatePercents(healthyPods, podsData?.length)}
+              dataBarColor={'var(--sapIndicationColor_8)'}
+              remainingBarColor={'var(--sapIndicationColor_8b)'}
+              tooltip={{
+                content: t('cluster-overview.tooltips.healthy-pods', {
+                  value: healthyPods,
+                  max: podsData?.length,
+                }),
+                position: 'bottom',
+              }}
+            />
+          )}
+          {deploymentsData && (
+            <ProgressIndicatorWithPercentage
+              title={t('cluster-overview.statistics.healthy-deployments')}
+              value={calculatePercents(
+                healthyDeployments,
+                deploymentsData?.length,
+              )}
+              dataBarColor={'var(--sapIndicationColor_6)'}
+              remainingBarColor={'var(--sapIndicationColor_6b)'}
+              tooltip={{
+                content: t('cluster-overview.tooltips.healthy-deployments', {
+                  value: healthyDeployments,
+                  max: deploymentsData?.length,
+                }),
+                position: 'bottom',
+              }}
+            />
+          )}
+        </CardWithTooltip>
+      </div>
+      <div
```

Comment:
if `data` `podsData` and `deploymentsData` will be unavailable then we will have additional margin with empty section